src/Microsoft.Extensions.Logging.Console/ConsoleLoggerProvider.cs

```diff
@@ -45,20 +45,32 @@ public ConsoleLoggerProvider(IConsoleLoggerSettings settings)
 
         private void OnConfigurationReload(object state)
         {
-            // The settings object needs to change here, because the old one is probably holding on
-            // to an old change token.
-            _settings = _settings.Reload();
+            try
+            {
+                // The settings object needs to change here, because the old one is probably holding on
+                // to an old change token.
+                _settings = _settings.Reload();
 
-            foreach (var logger in _loggers.Values)
+                foreach (var logger in _loggers.Values)
+                {
+                    logger.Filter = GetFilter(logger.Name, _settings);
+                    logger.IncludeScopes = _settings.IncludeScopes;
+                }
+            }
+            catch (Exception ex)
             {
-                logger.Filter = GetFilter(logger.Name, _settings);
-                logger.IncludeScopes = _settings.IncludeScopes;
+                foreach (var logger in _loggers.Values)
+                {
+                    logger.LogError(0, ex, "Error while loading configuration changes.");
```

Comment:
@pakrym I think it is perfectly fine to log an exception if the configuration reload failed. 
I guess it would just be cleaner to create a logger for the LoggerProvider itself. Logging an exception on all attached loggers is definitely not a good idea, there could be quite a few right?